nav/dataset/dataset_pds3_cassini_iss.py

```
@@ -1,6 +1,8 @@
 import argparse
-from typing import Any, Optional
+from pathlib import Path
+from typing import Any, Iterator, Optional, cast
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)